tests/unit_tests/test_wave_06.py

```diff
@@ -109,7 +118,13 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    
+    assert result == True
+    assert len(tai.inventory) == len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_c not in tai.inventory
```

Comment:
I recommend checking all of the items here, rather than just the ones that were swapped.